.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules
-yarn.lock
+yarn.lock
```

Comment:
This file should be unchanged

Reply:
Reverted at (#4eef206)[https://github.com/sindresorhus/public-ip/pull/59/commits/4eef20686de214e165ffeff0a0d5816b08b0315a]